basics.js

```diff
@@ -4,11 +4,17 @@
 
 var pets = ['cat', 'dog', 'wombat'];
 
+
+for each (pets) {
```

Comment:
Refer to the MDN for how to do for loops in JavaScript. This syntax doesn't work.